.github/workflows/swift.yml

```diff
@@ -111,4 +111,19 @@ jobs:
       - uses: actions/upload-artifact@v3
         with:
           name: swift-generated-cpp-files
-          path: swift/generated-cpp-files/**
\ No newline at end of file
+          path: swift/generated-cpp-files/**
+  qlformat:
```

Comment:
Hmm, this is most likely a rebase artifacts, not sure where it comes 🤔

Reply:
It's just a new line missing at the end of the file before this change. This adds a new line, so 114 changes.

Reply:
Assuming you also wanted to add a formatting check.

Reply:
Yeah, this is indeed a rebase artifact, the check was removed here https://github.com/github/codeql/pull/11214